contrib/babelfishpg_tsql/sql/object_definition_tsql.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE sys.babelfish_view_def (
```

Comment:
I would suggest this table to have only oid (pkey) and view definition columns. Otherwise, we need additional special handlings for `ALTER DATABASE` and `ALTER SCHEMA`.

Reply:
The problem with having oid as primary key is that oid might get changed during Major Version Upgrade/Pg dump-restore, in that case we've to figure out the oid and update the catalog which will be more complex. So it is better to have combined key of db, schema and object name and handle the cases for `ALTER DATABASE` and `ALTER SCHEMA`.

Reply:
Okay, that makes sense. I'm not sure whether all those functionalities are in the current BBF. If not, let's add at least test cases to notify future developers :) FYI, object name may also be changed by `sp_rename`.

If we are storing those names, I'd suggest to avoid using pg-only types because it can cause different result for comparisons. I guess T-SQL has their own types for db/schema/object names.